### PR TITLE
Embedding sbert class

### PR DESCRIPTION
**Pull request by elliegarcia**
Created a wrapper class for the SBERT embeddings. Please especially look over the hyperparameters in the SentenceTransformerEmbedding class

### CODE REVIEW

File: experiments/embedding_methods/sbert_embedding.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        """
+        converts a sentence into SBERT embeddings using the Sentence Transformers package
+        ----------
+        sentence : str
+            the string that is to be embedded
+        """
+        # Sentences are encoded by calling model.encode()
+        embedding = self.model.encode(sentence)
+        return embedding
+    def return_hyper_params(self):
```
Comment (samuelgoodall) [inline experiments/embedding_methods/sbert_embedding.py:34]: add docstrings documentation: 
https://numpydoc.readthedocs.io/en/latest/format.html
https://realpython.com/documenting-python-code/#docstring-formats

File: experiments/embedding_methods/sbert_embedding.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+
+class SentenceTransformerEmbedding(EmbeddingInterface):
+
+    def __init__(self, model_name: str):
+        self.model = SentenceTransformer(model_name)
+        self.model_name = model_name
+
+    def embed_sentence(self, sentence: str):
+        """
```
Comment (samuelgoodall) [inline experiments/embedding_methods/sbert_embedding.py:24]: Use Numpy docstring style:
https://numpydoc.readthedocs.io/en/latest/format.html
example is in the glove embedding class

**Review by samuelgoodall — CHANGES_REQUESTED** [review]
Looks good just minor changes for documentation style

File: experiments/embedding_methods/sbert_embedding.py (hunk 2 above)
Comment (samuelgoodall) [reply experiments/embedding_methods/sbert_embedding.py:24]: make sure the model that we use is getting the correct input-> does it have to be uncased?

Comment (samuelgoodall) [reply experiments/embedding_methods/sbert_embedding.py:24]: well have to make sure we dont cutoff long sentences like ansgar mentioned in the meeting

File: experiments/embedding_methods/sbert_embedding.py (hunk 1 above)
Comment (elliegarcia) [reply experiments/embedding_methods/sbert_embedding.py:34]: I added more notation to the sbert wrapper class, could you look over it?

File: experiments/embedding_methods/sbert_embedding.py (hunk 2 above)
Comment (elliegarcia) [reply experiments/embedding_methods/sbert_embedding.py:24]: As I understand the uncased model strips the casing of the words, I think we don't need to feed it uncased sentences.
Added the 512 cutoff.

**Pull request merged**